docs/DeveloperGuide.md

```diff
@@ -568,10 +538,8 @@ Priorities: High (must have) - `****`, Medium (nice to have) - `***`, Low (unlik
 
     Use case ends.
 
-<br/><br/>
-**System: ContactMate**      
-**Use case: UC08 \- Mark an elderly as called, given that elderly's name**      
-**Actor: Staff**        
+<br/><br/> 
+**Use case: UC08 \- Mark an elderly as called, given that elderly's name**       \
```

Comment:
I just realised that should we remove `given that elderly's name`, cuz we are using NRIC or index.

Reply:
The use case for that is if someone only has that elderly's name, so they have to search for that elderly (`find`), before marking by index or smth